### PR TITLE
Adding all methods to README.

### PR DESCRIPTION
**Pull request by moizjv**
- Part of clean up task

### CODE REVIEW

**Comment by moizjv** [comment]
Also updating dependencies

**Comment by imurchie** [comment]
There are other dependencies that can be updated. "appium-logger": "^2.0.0", "appium-support": "^2.0.0". "babel-runtime" should be locked at  "=5.8.24". And "rimraf" is unneeded ("appium support" has `fs.rimraf`).

File: lib/utils.js [hunk 1]
```diff
@@ -1,25 +1,9 @@
...
-      },
-      rootDir = path.resolve(__dirname, process.env.NO_PRECOMPILE ? '..' : '../..'),
+const rootDir = path.resolve(__dirname, process.env.NO_PRECOMPILE ? '..' : '../..'),
       androidPlatforms = ['android-4.2', 'android-17', 'android-4.3', 'android-18',
                           'android-4.4', 'android-19', 'android-L', 'android-20',
                           'android-5.0', 'android-21', 'android-22', 'android-MNC',
                           'android-23', 'android-6.0'];
 
-export { mkdirp, mv, rimraf, ncp, fs , rootDir, androidPlatforms };
+export { rootDir, androidPlatforms };
```
Comment (imurchie) [inline lib/utils.js]: Are these now "utilities"? Or should this file be renamed in order to reflect what it is now doing?

File: package.json [hunk 2]
```diff
@@ -28,22 +28,19 @@
...
-    "appium-support": "1.1.0",
+    "appium-support": "^2.0.1",
     "asyncbox": "^2.0.2",
     "babel-runtime": "=5.8.24",
     "bluebird": "^2.9.26",
     "lodash": "^3.9.1",
-    "mkdirp": "~0.5.0",
-    "mv": "~2.0.0",
-    "ncp": "~2.0.0",
     "rimraf": "~2.3.4",
```
Comment (imurchie) [inline package.json]: Shouldn't need.

**Pull request merged**